changelog/changelog.py

```diff
@@ -78,19 +78,39 @@ def callback(response, *args, **kwargs):
             hooks={'response': callback}
         )
 
-    def get_latest_milestone(self):
+    def fetch_open_milestones(self):
         """
-        Get info about the latest milestone currently open in issues.
+        Fetch list of milestone dicts. Each dict contains metadata about an open milestone.
         """
         url = f"{self.api_url_prefix}/repos/{self.repo_url}/milestones"
-        r = self.request(url).json()
-
-        logger.debug(f"Open milestones found: {r}")
-        if len(r) == 0:
+        open_milestones = self.request(url).json()
+        if not open_milestones:
             raise ValueError("No open milestone was found on github.")
-        elif len(r) > 1:
-            logger.info(f"Found multiple open milestones on github. Using latest: {r[0]}")
-        return r[0]
+        logger.debug(f"Open milestones found: {open_milestones}")
+        return open_milestones
+
+    def get_last_milestone(self, sort_key: str = 'updated_at'):
+        """
+        Get last open milestone from sorted list. By default, the most recently updated milestone is returned.
+        """
+        open_milestones = self.fetch_open_milestones()
+        milestone = sorted(open_milestones, key=lambda m: m[sort_key])[-1]
+        logger.info(f"Using milestone '{milestone['title']}', as sorted by '{sort_key}' attribute.")
+
+        return milestone
```

Comment:
I figure that sorting by `'updated_at'` is a more sensible default, given that the most recently updated milestone is likely the one that someone would want to generate a changelog for. (Plus, it's trivially easy to fix the order if it's incorrect, because even just editing the milestone's description will change the value of `'updated_at'`, causing it to be sorted first.)